### ISSUE TITLE
oim loss is not contain  CQ(circuler queue) softmax loss ?

### ISSUE THREAD

**Issue by HenryZhangJianhe** [serend1p1ty/SeqNet#8]
# 74  in oim.py
loss_oim=F.cross_entropy(projected,label,ignore_index=5554)
I have a question, if you ignore_index 5554, you just calculate labelled pids classification loss.
And where is unlabelled pids softmax cross_entorpy loss?
![image](https://user-images.githubusercontent.com/52202915/119105438-e5789600-ba4f-11eb-9301-f0f1ad5253bf.png)

**Comment by serend1p1ty** [comment]
Label info is needed by cross entropy loss, so the loss is only for labelled people.

**Comment by HenryZhangJianhe** [comment]
Do you mean there is no loss for i-th unlabelled identity in the circular queue in your code like eqt(2) of paper 'joint detection and identification for person search'

**Comment by serend1p1ty** [comment]
Unlabelled people are used in OIM-circular queue.
https://github.com/serend1p1ty/SeqNet/blob/58471df02baacd338a53037e8856bc05b2be2229/models/oim.py#L33
They don't participate in the calculation of cross entropy loss.
